lib/tree-view.coffee

```diff
@@ -869,7 +869,7 @@ class TreeView
       return
 
     entryName = path.basename(initialPath)
-    newPath = "#{newDirectoryPath}/#{entryName}".replace(/\s+$/, '')
+    newPath = path.join(newDirectoryPath, entryName).replace(/\s+$/, '')
```

Comment:
I also don't think the replace is necessary.  I haven't seen any other case where we trim paths.